states.go

```diff
@@ -13,7 +13,8 @@ const (
 	TaskState_TASK_LOST     TaskState = "TASK_LOST"
 	TaskState_TASK_ERROR    TaskState = "TASK_ERROR"
 	// Custom eremetic states
-	TaskState_TASK_QUEUED TaskState = "TASK_QUEUED"
+	TaskState_TASK_QUEUED      TaskState = "TASK_QUEUED"
+	TaskState_TASK_TERMINATING TaskState = "TASK_TERMINATING"
```

Comment:
Agreed that we should keep the naming consistent.

I believe it will become `TASK_KILLED` but we should test that.